Simplify the foxfire card, And Replace name with swarms card (#1864)
crawl/crawl: crawl-ref/source/dat/descript/cards.txt

Drawing this card will summon a dancing weapon. Its quality, brand, and
friendliness are all affected by card power.
%%%%

> the Swarms card

Reviewer: ```suggestion
the Swarm card
```

> The card depicts a swarm of midday bees filled with humming noises.

Reviewer: ```suggestion
The card depicts a bear surrounded by a swarm of bees. It hums ominously.
```

> Drawing this card will summon forth bees swarms.

Reviewer: ```suggestion
Drawing this card will summon bees.
```
